src/Integration.Vsix/Analysis/SingleFileMonitor.cs

```diff
@@ -73,7 +73,7 @@ public SingleFileMonitor(IFileSystemWatcherFactory factory, IDirectory directory
             fileWatcher.Changed += OnFileChanged;
             fileWatcher.Created += OnFileChanged;
             fileWatcher.Deleted += OnFileChanged;
-            fileWatcher.Renamed += (s, args) => OnFileChanged(s, args);
+            fileWatcher.Renamed += OnFileRenamed;
```

Comment:
Why not use OnFileChanged? (like the rest of the events)

Reply:
The event argument type is different.

Reply:
...but we could still use the same handler method. Not sure why it was ever different...
I'll use `OnFileRenamed`